chutney/action-impl/src/main/java/com/chutneytesting/action/function/MathFunctions.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * SPDX-FileCopyrightText: 2017-2024 Enedis
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package com.chutneytesting.action.function;
+
+import com.chutneytesting.action.spi.SpelFunction;
+
+public class MathFunctions {
```

Comment:
Why not to handle other numeric types (`Float`, `Long`, `Double`,...)

Reply:
So, for example, should I write 4 abs() functions for each type ?

Reply:
Or handle all type in the same function.
@nbrouand @boddissattva what do you think ?

Reply:
Number as input and manage type in the method. Test with primitive type You can create a test scenario in com.chutneytesting.acceptance.tests.engine